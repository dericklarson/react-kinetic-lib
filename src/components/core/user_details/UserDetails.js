import React from 'react';

const UserDetails = props => <div className="user-details">User Details</div>;
UserDetails.propTypes = {};

export default UserDetails;
