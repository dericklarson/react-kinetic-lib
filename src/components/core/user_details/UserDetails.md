## User Details View

```js
<UserDetails />
```
