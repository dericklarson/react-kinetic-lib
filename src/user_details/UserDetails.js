import React from 'react';

export const UserDetails = props => (
  <div className="user-details">User Details</div>
);
