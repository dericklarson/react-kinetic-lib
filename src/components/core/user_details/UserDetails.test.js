import React from 'react';
import { shallow } from 'enzyme';

import UserDetails from './UserDetails';

describe('<UserDetails />', () => {
  it('should be sane', () => {
    expect(false).not.toBe(true);
  });

  it('should render correctly', () => {
    const component = shallow(<UserDetails />);

    expect(component).toMatchSnapshot();
  });
});
