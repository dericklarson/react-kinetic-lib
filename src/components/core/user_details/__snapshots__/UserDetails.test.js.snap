// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserDetails /> should render correctly 1`] = `
<div
  className="user-details"
>
  User Details
</div>
`;
